docker/fluent/Dockerfile

```diff
@@ -1,23 +1,56 @@
-ARG baseImage=centos:7
-FROM ${baseImage} as fluentbuilder
-COPY fluent.tgz /ansys_inc/
-RUN tar -xzf /ansys_inc/fluent.tgz --directory /ansys_inc/ && rm -f /ansys_inc/fluent.tgz
-
-FROM ${baseImage} as ceibuilder
-COPY cei.tgz /ansys_inc/
-RUN tar -xzf /ansys_inc/cei.tgz --directory /ansys_inc/ && rm -f /ansys_inc/cei.tgz
-
-FROM ${baseImage}
-LABEL description="Ansys Fluent 2022 R2"
-MAINTAINER Laurence Janoo "laurence.janoo@ansys.com"
-RUN yum install -y \
-        tcsh.x86_64 \
-        libXrandr.x86_64 \
-        libXinerama.x86_64 \
-        libXcursor.x86_64 \
-        mesa-libGLU-9.0.0-4.el7.x86_64 \
-        libglvnd-glx-1.0.1-0.8.git5baa1e5.el7.x86_64 \
-        libglvnd-1.0.1-0.8.git5baa1e5.el7.x86_64  && yum clean all
-COPY --from=fluentbuilder /ansys_inc /ansys_inc/
-COPY --from=ceibuilder /ansys_inc /ansys_inc/
-ENTRYPOINT ["/ansys_inc/v222/fluent/bin/fluent"]
+# Specify base image
+FROM ubuntu:20.04
+
+# User variables
+ARG USERNAME=fluent
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Create the user
+RUN groupadd --gid $USER_GID $USERNAME \
+    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME
+
+# Copy ``ansys_inc`` directory
+COPY ansys_inc /ansys_inc
+
+# Add metadata
+LABEL description="Ansys Fluent 2025 R1"
+LABEL email="pyansys.core@ansys.com"
+
+# OCI label
+LABEL org.opencontainers.image.documentation="https://fluent.docs.pyansys.com"
+
+# Install dependencies
+RUN apt-get -y update && apt-get -y install \
+        csh \
+        fontconfig \
+        libxi-dev \
+        libsm6 \
+        libice-dev \
+        libxrandr-dev \
+        libxinerama-dev \
+        libxcursor-dev \
+        libglu1-mesa \
+        libglvnd-dev  && apt-get clean all
+
+# Specify working directory
+ENV WORKING_DIRECTORY=/workdir
+
+# Specify home directory
+ENV HOME=/home/$USERNAME
+
+# Add a working directory and make it accessible to any user
+RUN mkdir -p /home/$USERNAME/$WORKING_DIRECTORY \
```

Comment:
Maybe create a single env var for the full working directory and use it in this command.

Reply:
Done. Thank you.